src/server/qtquick/woutputrenderwindow.cpp

```diff
@@ -1067,8 +1072,9 @@ bool OutputHelper::tryToHardwareCursor(const LayerData *layer)
                           ? layer->renderer->lastBuffer()->handle()
                           : nullptr;
         if (!buffer) {
-            if (set_cursor)
+            if (set_cursor && m_cursorRenderComplete)
```

Comment:
Don't care for `m_cursorRenderComplete` in this `if`.

Reply:
done